src/io/flutter/ProjectOpenActivity.java

```diff
@@ -55,6 +55,9 @@ public void runActivity(@NotNull Project project) {
       return;
     }
 
+    sdk.queryFlutterConfig("android-sdk", false);
```

Comment:
This will delay the items after this in runActivity() method. I don't know if they're time sensitive, or if the return from runActivity() is (if slowing it down will impact other things).

Perhaps, run these two queries on a separate thread via something like `JobScheduler.getScheduler().schedule(...)`?

Reply:
Finding the Android SDK happens later during project opening, so it isn't needed here. Finding Android Studio can certainly happen in the background, so I changed that.